roles/letsencrypt/tasks/http-challenge-local.yml

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
We can test this challenge in the CI, right?

Reply:
yep, works now :) 3a7dbbb